README.md

```diff
@@ -60,7 +60,7 @@ GitHub Apps have distinct identities on GitHub – no seat taken up by a machine
 
 ### Creating a GitHub App
 
-Here we are using a GitHub App as an authentication entity, rather than a traditional coding application. Below are some brief instructions on how to setup a GitHub App for this purpose, note that there are other methods of creating a GitHub App such as with a manifest file. _(The instructions below are for setting up an app within your user, but you can also do it for your organisation.)_
+Here we are using a GitHub App as an authentication entity. No code will be added to the GitHub App, instead we will be using its permissions when we execute our GitHub Workflows. rather than a traditional coding application. Below are some brief instructions on how to setup a GitHub App for this purpose, note that there are other methods of creating a GitHub App such as with a manifest file (e.g. one similar to [`app.yml`](/app.yml)). _(The instructions below are for setting up an app within your user, but you can also do it for your organization.)_
```

Comment:
I'd prefer to leave this sentence out. A GitHub App is a separate entity from an authentication standpoint but never itself has code "added to it".

```suggestion
Here we are using a GitHub App as an authentication entity. Below are some brief instructions on how to setup a GitHub App for this purpose, note that there are other methods of creating a GitHub App such as with a manifest file (e.g. one similar to [`app.yml`](/app.yml)). _(The instructions below are for setting up an app within your user, but you can also do it for your organization.)_
```